pom.xml

```diff
@@ -258,6 +258,7 @@
     <profile>
       <id>full-build</id>
       <modules>
+        <module>zookeeper-it</module>
```

Comment:
What the reason of this change? It seems a change we should cheery pick for master.

Reply:
Because fatjar needs it.
master branch is differentf from branch-3.6